wdfn-server/waterdata/views.py

```diff
@@ -115,6 +116,11 @@ def monitoring_location(site_no):
                 'questions_link': questions_link,
                 'cooperators': cooperators
             }
+
+            monitoring_camera = get_monitoring_camera_data(site_no)
+            if monitoring_camera:
```

Comment:
I'm not sure this `if` statement is necessary as currently written. `get_monitoring_camera_data` will always return a dictionary with some content, so it'll always evaluate to `True`.

Reply:
Right, this is because originally I did not return a dictionary and later I found that is was better to return one with the exists state. It could be removed.